eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -350,6 +359,30 @@ object Helpers {
         data.commitments.remoteCommit.spec.toRemoteMsat == 0 &&
         data.commitments.remoteNextCommitInfo.isRight
 
+    /**
+      * Check if a channel is closed (i.e. its closing tx has been confirmed)
+      * @param data channel state data
+      * @return the channel closing type, if any
+      */
+    def closingType(data: HasCommitments): Option[ClosingType] = data match {
```

Comment:
The function name isn't explicit enough, given that the primary purpose of this is to tell whether or not a channel can be considered closed.

```suggestion
    def isClosed(data: HasCommitments): Option[ClosingType] = data match {
```